contracts/DomainRoles.sol

```diff
@@ -15,7 +15,7 @@
   along with The Colony Network. If not, see <http://www.gnu.org/licenses/>.
 */
 
-pragma solidity 0.5.8;
+pragma solidity 0.5.8; // ignore-swc-103
```

Comment:
Hmm, these look locked to me? I don't see any carets (`^`).

Reply:
Yes, again bad `truffle-security` warning that isn't present in v 1.5.x https://circleci.com/gh/JoinColony/colonyNetwork/8796